src/internal/pfsdb/commits.go

```diff
@@ -726,6 +726,15 @@ func getCommitInfoFromCommitRow(ctx context.Context, extCtx sqlx.ExtContext, row
 	if err != nil {
 		return nil, errors.Wrap(err, "get provenance for commit")
 	}
+	subvenantCommits, err := getCommitSubvenance(ctx, extCtx, row.ID)
```

Comment:
I think we are going to need to set the depth option to 1 here.

Reply:
Adding some tests for the direct provenance / subvenance would be helpful.

Reply:
Yup, you're right. I'm not sure why that slipped my mind.